dipy/core/gradients.py

```diff
@@ -1343,3 +1343,133 @@ def _check_ornt(ornt):
     if tuple(uniq) not in {(-1, 1), (-1,), (1,)}:
         print(tuple(uniq))
         return True
+
+
+def extract_b0(dwi, b0_mask, *, group_contiguous_b0=False, strategy="mean"):
+    """Extract a set of b0 volumes from a dwi dataset.
+
+    Parameters
+    ----------
+    dwi: ndarray
+        4D diffusion-weighted imaging data (X, Y, Z, N), where N is the number of
+        volumes.
+    b0_mask: ndarray of bool
+        Mask over the time dimension (4th) identifying b0 volumes.
+    group_contiguous_b0: bool, optional
+        If True, each contiguous b0 volumes are grouped together.
+    strategy: str, optional
```

Comment:
Please mention what is default value

Reply:
No, please don't: @maharshi-gor default values are documented in the function signature, and adding them to the docstrings only increases the maintenance burden and the risk of going out of sync if the default value changes. So unless the default values triggers some secondary computation that needs to be documented, please do not add default values to docstrings. This has been mentioned multiple times.

What may be relevant is to mention the different possible strategies, either here or in the function kong description.